src/test/java/io/vertx/gsoc2018/qotd/QuoteOfTheDayVerticleTest.java

```diff
@@ -61,11 +61,10 @@ public void testRealtimeWebSocket(TestContext testContext) throws InterruptedExc
       });
     });
 
-    // wait some time for websocket connection establishing and subscription availability
-    Thread.sleep(1000);
-
     // force receiving a update
-    postQuote(SAMPLE_AUTHOR_NAME, SAMPLE_QUOTE_TEXT, testContext);
+    for (int i = 0; i < 100; i++) {
```

Comment:
This will work most often but is not bullet proof. Try to find a way to start posting when the websocket has been established

Reply:
ha, you did it in the next commit :+1: